src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/backup.yml

```diff
@@ -0,0 +1,41 @@
+admin_backup:
+    path: /
+    methods: [GET]
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\AdvancedParameters\Backup:index'
+        _legacy_controller: 'AdminBackup'
+
+admin_backup_process:
+    path: /
+    methods: [POST]
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\AdvancedParameters\Backup:processForm'
+        _legacy_controller: 'AdminBackup'
+
+admin_backup_create:
+    path: /create
+    methods: [POST]
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\AdvancedParameters\Backup:processBackupCreate'
+        _legacy_controller: 'AdminBackup'
+
+admin_backup_view_download:
+    path: /view/{downloadFileName}
+    methods: [GET]
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\AdvancedParameters\Backup:downloadView'
+        _legacy_controller: 'AdminBackup'
+
+admin_backup_delete:
+    path: /delete/{deleteFileName}
```

Comment:
would you mind to use POST method for this one? The best is using DELETE, I'll document how to do it soon.

Reply:
I had a feeling you will ask for it, and of course it makes sense. :)

Done with DELETE. :+1: